### PR TITLE
Portland blog post - schedule, tickets, live streaming

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

**Comment by thatdocslady** [comment]
Something is broken and I can't figure it out... help?

File: docs/conf/portland/2019/news/schedule-tickets.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+   :tags: 2019, portland-2019, schedule, tickets
+
+Announcing Full Schedule, Live Streaming, and Tickets Sold Out
+==============================================================
+
+Hello everyone!
+We're about a month out from the Portland conference and we hope you're getting ready!
+We're writing today to announce the full presentation schedule and to give you the latest registration update.
+
+In the meantime, if it's your first Write the Docs, now's a great time to check out `the Welcome Wagon page <https://www.writethedocs.org/conf/portland/2019/welcome-wagon/>`_, which is chockfull of info to help you get oriented.
```
Comment (plaindocs) [inline docs/conf/portland/2019/news/schedule-tickets.rst]:
```suggestion
In the meantime, if it's your first Write the Docs, now's a great time to check out `the Welcome Wagon page <https://www.writethedocs.org/conf/portland/2019/welcome-wagon/>`_, which is packed with info to help you get oriented.
```

File: docs/conf/portland/2019/news/schedule-tickets.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+And whether it's your first or fifth conference, it's never too early to check out the `Explore Portland
+<https://www.writethedocs.org/conf/portland/2019/visiting/>`_ page for inspiration on how to make the most of your visit.
+
+Tickets are sold out
+--------------------
+
+Every year we see more enthusiasm from the community, and we are happy and sad at the same time to announce that we are now **sold out** at capacity.
```
Comment (plaindocs) [inline docs/conf/portland/2019/news/schedule-tickets.rst]:
```suggestion
Every year we see more enthusiasm from the community, and we are happy and sad at the same time to announce that we are now **sold out** at capacity.
```
We're not really sad. Can we figure out another way to say this?

Comment (plaindocs) [reply docs/conf/portland/2019/news/schedule-tickets.rst]: Sorry to disappoint those who don't have tickets yet maybe?

**Review by plaindocs — COMMENTED** [review]
LGTM

**Comment by plaindocs** [comment]
Looks like the datatemplates is unhappy. @ericholscher did you kick it to fix something over in another PR? 

@thatdocslady this now merges into the schedule branch, so we can merge this first when we're happy with the post.

File: docs/conf/portland/2019/news/schedule-tickets.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+We're writing today to announce the full presentation schedule and to give you the latest registration update.
+
+In the meantime, if it's your first Write the Docs, now's a great time to check out `the Welcome Wagon page <https://www.writethedocs.org/conf/portland/2019/welcome-wagon/>`_, which is packed with info to help you get oriented.
+And whether it's your first or fifth conference, it's never too early to check out the `Explore Portland
+<https://www.writethedocs.org/conf/portland/2019/visiting/>`_ page for inspiration on how to make the most of your visit.
+
+Tickets are sold out
+--------------------
+
+Every year we see more enthusiasm from the community, and we are happy and sad at the same time to announce that we are now **sold out** at capacity.
```
Comment (thatdocslady) [inline docs/conf/portland/2019/news/schedule-tickets.rst]:
```suggestion
Every year we see more enthusiasm from the community, and we are grateful for your support! Tickets are now **sold out** and we are at maximum capacity.
```

File: docs/conf/portland/2019/news/schedule-tickets.rst (hunk 2 above)
Comment (thatdocslady) [reply docs/conf/portland/2019/news/schedule-tickets.rst]: made a suggestion?

**Pull request merged**